<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SongItem</class>
 <widget class="QWidget" name="SongItem">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>32</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>460</width>
    <height>32</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>32</height>
   </size>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1,0,0">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="buttonPlay">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Play</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonFadeIn">
     <property name="text">
      <string>FI</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelSongName">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>226</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>226</width>
       <height>20</height>
      </size>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="toolTipDuration">
      <number>1</number>
     </property>
     <property name="text">
      <string>Название песни</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonFadeOut">
     <property name="text">
      <string>FO</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonRepeat">
     <property name="text">
      <string>Repe</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
